spec/factories/search_result.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+class SearchResult < Hashie::Mash
```

Comment:
This is just a workaround to get the JSON response values working as expected for the factory.